doc/guide/packages.xml

```diff
@@ -186,6 +186,12 @@ $ cockpit-bridge --packages
         <listitem><para>The relative path to the HTML file within the package that implements
           the menu item or tool.</para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>docs</term>
+        <listitem><para>This item contains a list of documentation URLs for the given page.
```

Comment:
The "This item contains a" feels redundant -- the whole document is about that, after all. :wink: